lib/disco_log/integrations/plug.ex

```diff
@@ -76,24 +76,27 @@ defmodule DiscoLog.Integrations.Plug do
 
       def call(conn, opts) do
         unquote(__MODULE__).set_context(conn)
-        super(conn, opts)
-      rescue
-        e in Plug.Conn.WrapperError ->
-          unquote(__MODULE__).report_error(e.conn, e.reason, e.stack, config())
 
-          Conn.WrapperError.reraise(e)
-
-        e ->
-          stack = __STACKTRACE__
-          unquote(__MODULE__).report_error(conn, e, stack, config())
-
-          :erlang.raise(:error, e, stack)
-      catch
-        kind, reason ->
-          stack = __STACKTRACE__
-          unquote(__MODULE__).report_error(conn, {kind, reason}, stack, config())
-
-          :erlang.raise(kind, reason, stack)
+        case conn do
+          %{adapter: {Plug.Cowboy.Conn, _}} ->
```

Comment:
Cowboy is a dev dependency so I think this will error on a project without cowboy instead replace with a guard.
```
when is_struct(conn, Plug.Cowboy.Conn)
```

Reply:
I tested it with an empty project and it seems to compile just fine? The difference with https://github.com/mrdotb/disco-log/commit/8fefc40de746ce6cfee33c3841a1a03a3b0a5b65 is that there I matched on struct, which compiler expected to have, while here it's a module name which is just an atom.